schema.sql

```diff
@@ -1,5 +1,67 @@
 /* Database schema to keep the structure of entire database. */
```

Comment:
Great job team 🤝

- [ ] As stated in the project requirements,  new tables should be created in the `schema_based_on_diagram.sql` file instead of the `schema.sql` file. Kindly make this change and resubmit your work.